tabml/model_analysis.py

```diff
@@ -91,9 +91,29 @@ def __init__(
     def analyze(self):
         if self.show_feature_importance:
             self._show_feature_importance()
+        if hasattr(self.model_wrapper, "sample_weights"):
+            self._save_sample_weight()
         self._analyze_metrics_one_dataset(self.train_dataset_name)
         self._analyze_metrics_one_dataset(self.val_dataset_name)
 
+    def _save_sample_weight(self):
+        save_path = Path(self.output_dir) / "sample_weight.csv"
+        feature_to_create_weights = self.data_loader.feature_to_create_weights
+        feature_values = self.data_loader.feature_manager.extract_dataframe(
+            features_to_select=feature_to_create_weights,
```

Comment:
`features_to_select` is a list while `feature_to_create_weights` is a string. Could you double check?